src/arch/xtensa/gdb/init.S

```diff
@@ -24,7 +24,7 @@ gdb_init_debug_exception:
 
 	/* enable breakpoints */
 	movi a3, 1
-	wsr  a3, IBREAKENABLE
+	wsr  a3, DEBUG_IBREAKENABLE
```

Comment:
Why is it changed? Also, did you try build with config for gdb debug enabled?

Reply:
It's changed, because IBREAKANABLE is in one of xtensa headers and I wanted to avoid inclusion, since we've already redefined it to DEBUG_IBREAKANABLE to make it available from outside of the arch.
Yes, last time I checked it worked.